src/catalog/protos/objects.proto

```diff
@@ -694,7 +695,7 @@ message AuditLogEventV1 {
     RenameSchemaV1 rename_schema_v1 = 27;
     UpdateItemV1 update_item_v1 = 26;
     CreateSourceSinkV3 create_source_sink_v3 = 29;
-        AlterRetainHistoryV1 alter_retain_history_v1 = 30;
+    AlterRetainHistoryV1 alter_retain_history_v1 = 30;
```

Comment:
I guess we don't lint our .proto files? I'm tempted to just leave this all in because why not.